reports-backend/src/main/java/org/jboss/da/reports/api/SCMLocator.java

```diff
@@ -19,6 +22,14 @@
 
     @Getter
     @NonNull
+    private Set<Long> productIds = new HashSet<>();
+
+    @Getter
+    @NonNull
+    private Set<Long> productVersionIds = new HashSet<>();
```

Comment:
I don't think these belong to SCMLocator.

I think in SCMLocator should be only data necessary to clone and analyse SCM repository (scmUrl and revision are obvoious, pomPath is path to pom that should be analysed and repositories is list of optional repositories that are needed for the analysis).